### PR TITLE
docs(router): what value to assign to 'routerLink' attribute of anchor tag in order to render a component.

### PR DESCRIPTION
**Pull request by sumitrawat10**
docs(router): what value to assign to 'routerLink' attribute of anchor tag in order to render a component.

 It was not clear to me what exact value do we need to assign to `routerLink' attribute of an anchor tag in order to render the component

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by sumitrawat10** [comment]
I have provided the CLA and have done a rescan from the CLA page but still that CLA setup is failing. Any hints.

File: aio/content/guide/router.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This topic describes how to implement many of the common tasks associated with a
 ## Generate an application with routing enabled
 
 The following command uses the Angular CLI to generate a basic Angular application with an application routing module, called `AppRoutingModule`, which is an NgModule where you can configure your routes.
-The application name in the following example is `routing-app`.
+The application name in the following example is `routing-/app`.
```
Comment (JeanMeche) [inline aio/content/guide/router.md:10]: Can you ammend your commit to remove this ?

**Comment by JeanMeche** [comment]
@sumitrawat10 Did you make the change with the github UI ? The mail adress behind the commit is `47861908+sumitrawat10​@users.noreply.github.com`. You should amend the commit to fix the email adress.

**Comment by JeanMeche** [comment]
Hi, we're looking at deprecating angular.io in favor of https://angular.dev. 

As your PR has curently some conflict i'm going to close it. Feel free to drop any PR for improvements on the new doc site, thank you.

**Pull request closed without merging**